src/components/metric-chart.component.ts

```diff
@@ -59,14 +64,25 @@ export class MetricChartComponent implements OnChanges {
   @Input() timestampLabel = 'Timestamp';
   @Input() rawData?: NumericDataPoint[];
   @Input() statsData?: NumericBucketPoint[];
+  @Input() multiData: IMultiDataPoint[];
   @Input() forecastDataPoints: IPredictiveMetric[];
-  @Input() showDataPoints = false;
+  @Input() showDataPoints = true;
   @Input() previousRangeData = [];
   @Input() annotationData = [];
   @Input() yAxisUnits: string;
-  @Input() multiData: IMultiDataPoint[];
   @Input() raw: false;
   @Input() buckets = 60;
+  @Output() timeRangeChange = new EventEmitter();
+  timeRangeValue: TimeRange = 43200;
+  @Input()
```

Comment:
I want two-ways binding on timeRange, and that's the way I found to do it (I read it from there: https://blog.thoughtram.io/angular/2016/10/13/two-way-data-binding-in-angular-2.html ). I'm definitely not comfortable yet with those "get" and "set" keywords :)